### PR TITLE
melodic -> master

### PR DESCRIPTION
**Pull request by mlautman**
### Description

Updating the master branch
### Checklist
- [x] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

**Comment by davetcoleman** [comment]
I just made master default

File: README.md [hunk 1]
```diff
@@ -16,17 +16,21 @@ This repository is currently built automatically by two systems. Travis builds t
...
 - ``indigo-devel`` usage is discouraged
 - ``kinetic-devel`` stable
-- ``melodic-devel`` latest, changes should target this branch
+- ``melodic-devel`` stable
+- ``master`` latest, changes should target this branch
 
 ## Build Locally
 
-If you want to test the tutorials by generating the html pages locally on your machine, use the following helper script.
+If you want to test the tutorials by generating the html pages locally on your machine, use the ``build_locally`` script.
```
Comment (davetcoleman) [inline README.md]:
```suggestion
If you want to test the tutorials by generating the html pages locally on your machine, use the ``build_locally.sh`` script.
```

File: conf.py [hunk 2]
```diff
@@ -18,9 +18,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = 'Kinetic'
+version = 'Melodic'
 # The full version, including alpha/beta/rc tags.
-release = 'Kinetic'
+release = 'Melodic'
```
Comment (davetcoleman) [inline conf.py]: will this cause 16.04 machines to no longer work?

File: index.rst [hunk 3]
```diff
@@ -8,7 +8,7 @@ These tutorials will step you through using and learning the MoveIt! Motion Plan
...
 
-.. note:: You are on the latest **ROS Melodic** version of the tutorials, which is less stable. For beginners we recommmend the more stable `ROS Kinetic tutorials <http://docs.ros.org/kinetic/api/moveit_tutorials/html/index.html>`_ (Requires Ubuntu 16.04). For older computers on Ubuntu 14.04 see `ROS Indigo tutorials with the PR2 <http://docs.ros.org/indigo/api/moveit_tutorials/html/doc/ikfast_tutorial.html>`_.
+.. note:: You are on the latest **master** version of the tutorials, which is less stable. For beginners we recommmend the more stable `ROS Kinetic tutorials <http://docs.ros.org/kinetic/api/moveit_tutorials/html/index.html>`_ (Requires Ubuntu 16.04). For older computers on Ubuntu 14.04 see `ROS Indigo tutorials with the PR2 <http://docs.ros.org/indigo/api/moveit_tutorials/html/>`_.
```
Comment (davetcoleman) [inline index.rst]: we should start recommending the ROS Melodic tutorials, but it seems they are not built yet @130s

File: conf.py (hunk 2 above)
Comment (mlautman) [reply conf.py]: My understanding is that this is only used in the html <title> tag. It was last changed here:
https://github.com/ros-planning/moveit_tutorials/pull/141

File: index.rst (hunk 3 above)
Comment (mlautman) [reply index.rst]: I agree. Once they are being built again I will open a new PR

Comment (130s) [reply index.rst]: > we should start recommending the ROS Melodic tutorials, but it seems they are not built yet @130s

I think this is reported at https://github.com/ros-planning/moveit_tutorials/issues/293

**Pull request merged**